### PR TITLE
Update the Prerequisites steps to include "Destination Filters" 

### PR DESCRIPTION
**Pull request by prigiattiperrut**
### Proposed changes

Added a required step to the Prerequisites section, so customers are aware that enabling  "Destination Filters"  is necessary. 

<!-- When should this get merged/published?
- ASAP once approved

### Related issues (optional)

Slack: https://twilio.slack.com/archives/C0114BL66EP/p1718296707639409?thread_ts=1718191907.764089&cid=C0114BL66EP

### CODE REVIEW

**Comment by atitsh** [comment]
@prigiattiperrut @forstisabella , if i am reading the changes correct, i dont think it is in the suggested spot. I thought we wanted to add this to the Pre-req steps? no?

I think that is the right spot, plus we should call out, if using Analytics.js and Device Mode destination, enable destination filters.

File: src/privacy/consent-management/configure-consent-management.md [hunk 1]
```diff
@@ -24,10 +24,11 @@ Before you can configure consent in Segment, take the following steps:
 > info "Limited availability of destinations"
 > AWS S3 and Engage destinations do not enforce consent preferences. 
 
-1. From the [Segment homepage](https://app.segment.com/goto-my-workspace/){:target="_blank”}, select the Privacy tab and click **Consent Management**.
-2. On the Consent management page, click **Create categories**.
-3. Confirm that you have completed the required prerequisites, and click **Next**.
-4. On the Create consent categories page, add the following information to the category form:
+1. For Analytics.js sources only - On the Source Settings page, enable **Destination Filters**.
```
Comment (forstisabella) [inline src/privacy/consent-management/configure-consent-management.md:27]:
```suggestion
1. _For Analytics.js sources only_ : Navigate to your Analytics.js source and select  **Settings > Analytics.js**. Enable **Destination Filters**.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]
I agree with @atitsh - this seems like the right information, but is just in the wrong place. I think you could add this one of two places: 

- In the prerequisites section, as the last step
- In the section about wrappers, as a note under the link to the A.js wrapper link

**Comment by atitsh** [comment]
@forstisabella - i like that, lets move these changes to those 2 places and publish the doc. best to publish a patch now and not wait for next week's tuesday release train.

**Comment by forstisabella** [comment]
closing this PR, as the information was replicated in #6706. Thank you for catching this issue and drafting the language @prigiattiperrut!

**Pull request closed without merging**